includes/css/tec_default.css

```diff
@@ -288,7 +346,7 @@ button:not(.toggle).tec_dm_button {
     background-color: var(--ted-dgreen);
     padding-left: 5px;
     transition: all ease 0.5s;
-    opacity: 0;
+    opacity: 1;
```

Comment:
Prevented the words on the video embed header from appearing. This needed to be a temporary JS style instead of baked into the default stylesheet.